### PR TITLE
[GCP Cloudbuild plugin] Add Trigger Name

### PR DESCRIPTION
**Pull request by DaisukeHirabayashi**
## Hey, I just made a Pull Request!
With this change, the Trigger Name will be displayed.
Currently, it is not clear which Trigger has been activated, so we are making corrections.
CloudBuildPlugin

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by vinzscam — CHANGES_REQUESTED** [review]
nice!

File: .changeset/sour-snails-thank.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-cloudbuild': minor
+---
+
+Add Trigger Name
```
[the remark this answers is not in the document]
Comment (DaisukeHirabayashi) [reply .changeset/sour-snails-thank.md]: Thank you for suggesting, accepted ！

File: .changeset/sour-snails-thank.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-cloudbuild': minor
+---
+
+Added a new column to `EntityCloudbuildContent` displaying the trigger name for each build.
```
Comment (DaisukeHirabayashi) [inline .changeset/sour-snails-thank.md:5]:
```suggestion
Added a new column to `EntityCloudbuildContent` displaying the trigger name for each build.
```

**Comment by DaisukeHirabayashi** [comment]
I performed the sign-off and suggestion, but the DCO CI is not running. I tried to trigger the CI by adding a merge commit, but it did not work😢

**Comment by DaisukeHirabayashi** [comment]
@vinzscam 
When you have a moment, could you please review the changes again? 
I have signed all the commits.

**Review by vinzscam — APPROVED** [review]
let's :shipit:

**Comment by DaisukeHirabayashi** [comment]
DCO...

**Comment by vinzscam** [comment]
> DCO...

let me check...

**Comment by DaisukeHirabayashi** [comment]
@vinzscam 
I really appreciate your help! I can’t thank you enough.

**Comment by vinzscam** [comment]
> @vinzscam I really appreciate your help! Thank you for merging！

oh no problem, happy to help!

**Pull request merged**